### DISCUSSION TITLE
Requires area?

### DISCUSSION THREAD

**Discussion by nowayf** [TeamNewPipe/NewPipe#11452] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Cannot watch using Hong Kong nodes

**Comment by opusforlife2** [comment]
https://en.wikipedia.org/wiki/Censorship_of_YouTube#Countries_where_access_to_YouTube_is_currently_blocked

Not sure what "via a local version" means. That might have something to do with it.
